lineapy/db/db.py

```diff
@@ -619,7 +619,7 @@ def get_source_code_for_session(self, session_id: LineaID) -> str:
             )
 
     def delete_artifact_by_name(
-        self, artifact_name: str, version: Union[int, str] = None
+        self, artifact_name: str, *, version: Union[int, str]
```

Comment:
dont use the * or / syntax. it was introduce in 3.8+ and makes our code incompatible with python 3.7 runtime.

Reply:
simply removing the default should suffice for now.